docs/application/native/guides/connectivity/inm.md

```diff
@@ -0,0 +1,171 @@
+# INM
```

Comment:
This is a new page so kindly update the TOC file for its placement. Also as a suggestion to add a an description regarding INM on [Connectivity and Wireless overview](https://docs.tizen.org/application/native/guides/connectivity/overview) page.

Reply:
Hi, regarding TOC file update: would editing toc_all.md suffice, or would I need to edit the XML file too?

Reply:
@suraj-geddam Kindly update the `toc_all.md` file only. No need to edit the XML.

Reply:
@suraj-geddam Kindly update the title as suggested by @mijin85cho

Reply:
Oh sorry, forgot about this. It is changed now.